### PR TITLE
WIP 2019 design

### PR DESCRIPTION
**Pull request by okonet**
This is the WIP on the new design: https://www.figma.com/file/HRK5IaLG00uj7oEV2XOg45/Reason-Conf-2019?node-id=5%3A2

### CODE REVIEW

File: components/impressionsSection.re [hunk 1]
```diff
@@ -16,7 +16,10 @@ let make = _children => {
         <div className=style##layout>
           <h1 className=style##heading> {"Impressions from 2018" |> s} </h1>
           <div className=style##tweets>
-            <TwitterEmbed tweetId="995648995992178688" />
+            <TwitterEmbed
+              tweetId="995648995992178688"
+              options={TwitterEmbed.embedOptions(~theme="dark")}
```
Comment (okonet) [inline components/impressionsSection.re]: Ah, I was missing this part `TwitterEmbed.` 🤦‍♂️

File: data/content/pages/index.js [hunk 2]
```diff
@@ -31,6 +31,16 @@ const about = {
   title: "About"
 };
 
+const imprint = {
+  id: "imprint",
+  title: "Imprint"
```
Comment (okonet) [inline data/content/pages/index.js]: Can we have this for all pages and use this in the template?

File: pages/imprint.mdx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Imprint
+
+**Reason Association**<br />
+Verein zur Förderung der ReasonML & OCaml Plattform
+
+Zollergasse 2 / 18<br />
+1070 Vienna, Austria<br />
+hi@reason-conf.com
```
Comment (okonet) [inline pages/imprint.mdx]: This should be a link?

File: pages/imprint.mdx [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+https://www.reason-association.org/
```
Comment (okonet) [inline pages/imprint.mdx]: This should also be a link?

**Comment by ryyppy** [comment]
@okonet Everything running, styling is sometimes broken, but I would like starting merging and fixing issues later on

**Comment by okonet** [comment]
@ryyppy done!

**Pull request merged**